doc/source/train/getting-started-pytorch-lightning.rst

```diff
@@ -23,7 +23,7 @@ For reference, the final code is as follows:
     from ray.train.torch import TorchTrainer
     from ray.train import ScalingConfig
 
-    def train_func(config):
```

Comment:
Should we also make the same change for all the other places in the doc that show `train_func`?

Reply:
Done